sdk/storage/azure-storage-blob/tests/test_block_blob.py

```diff
@@ -82,19 +83,41 @@ def _create_source_blob(self, data):
     def _get_bearer_token_string(self, resource: str = "https://storage.azure.com/.default") -> str:
         return "Bearer " + f"{self.get_credential(BlobServiceClient).get_token(resource).token}"
 
+    def _build_file_share_headers(self, bearer_token_string: str, content_length: int = 0) -> Dict[str, Any]:
+        return {
+            'Accept': 'application/xml',
+            'Accept-Encoding': 'gzip, deflate',
+            'Authorization': bearer_token_string,
+            'Connection': 'keep-alive',
+            'Content-Length': str(content_length),
+            'x-ms-date': datetime.now(timezone.utc).strftime('%a, %d %b %Y %H:%M:%S GMT'),
+            'x-ms-version': get_api_version({}),
+            'x-ms-file-request-intent': 'backup',
+        }
+
     def _create_file_share_oauth(
-        self, storage_account_name: str,
+        self, bearer_token_string: str,
+        storage_account_name: str,
         data: bytes
-    ) -> Tuple[ShareServiceClient, ShareClient, ShareFileClient]:
-        share_service_client = ShareServiceClient(
-            account_url=self.account_url(storage_account_name, "file"),
-            credential=self.get_credential(ShareServiceClient),
-            token_intent='backup'
-        )
-        share_client = share_service_client.create_share(self.get_resource_name('utshare'))
-        file_client = share_client.get_file_client(self.get_resource_name('file'))
-        file_client.upload_file(data)
-        return share_service_client, share_client, file_client
+    ) -> Tuple[str, str]:
+        share_name = self.get_resource_name('utshare')
+        file_name = self.get_resource_name('file')
+        base_url = f"https://{storage_account_name}.file.core.windows.net/{share_name}"
+
+        # Creates file share
+        requests.put(url=base_url + "?restype=share", headers=self._build_file_share_headers(bearer_token_string))
```

Comment:
Requests also has the concept of a `Session` like you did for `aiohttp`. Could be nice to use that here for sync too.
https://requests.readthedocs.io/en/latest/user/advanced/